Dockerfile.dapper

```diff
@@ -13,32 +13,26 @@ ENV DAPPER_OUTPUT bin
 ENV DAPPER_SOURCE /go/src/github.com/longhorn/backing-image-manager
 WORKDIR ${DAPPER_SOURCE}
 
-ENV DEBIAN_FRONTEND=noninteractive
+
+RUN zypper -n addrepo --refresh https://download.opensuse.org/repositories/system:/snappy/SLE_15/system:snappy.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:tools:scm/SLE_15_SP3/devel:tools:scm.repo && \
+    zypper --gpg-auto-import-keys ref
 
 # Install packages
-RUN apt-get update && \
-    apt-get dist-upgrade -y && \
-    apt-get install -y cmake wget curl git less file \
-        libglib2.0-dev libkmod-dev libnl-genl-3-dev linux-libc-dev pkg-config psmisc tox qemu-utils fuse python-dev \
-        devscripts debhelper bash-completion librdmacm-dev libibverbs-dev xsltproc docbook-xsl \
-        libconfig-general-perl libaio-dev libc6-dev iptables libltdl7 libdevmapper-dev iproute2 jq
+RUN zypper -n install cmake wget curl git less file \
+        libkmod-devel libnl3-devel linux-glibc-devel pkg-config psmisc python3-tox fuse python3-devel \
+        bash-completion librdmacm1 librdmacm-utils libibverbs xsltproc docbook-xsl-stylesheets \
+        perl-Config-General libaio-devel glibc-devel-static glibc-devel iptables libltdl7 libdevmapper1_03 iproute2 jq docker gcc
 
 # needed for ${!var} substitution
 RUN rm -f /bin/sh && ln -s /bin/bash /bin/sh
 
 # Install Go & tools
 ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
-RUN wget -O - https://storage.googleapis.com/golang/go1.14.1.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
+RUN wget -O - https://storage.googleapis.com/golang/go1.13.15.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
```

Comment:
I've changed the golang version in Dockerfile.dapper to the latest v1.13.15 to match the version defined in go.mod